generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management.component.spec.ts.ejs

```diff
@@ -86,49 +81,49 @@ describe('Component Tests', () => {
 
     it('Should call load all on init', async () => {
       // GIVEN
-      mockedAxios.get.mockReturnValue(Promise.resolve({headers: {}, data: [{id: <%- tsKeyId %>}]}));
+      <%= entityInstance %>ServiceStub.retrieve.resolves({ headers: {}, data: [{ id: <%- tsKeyId %> }] });
 
       // WHEN
       comp.retrieveAll<%= entityAngularName %>s();
       await comp.$nextTick();
 
       // THEN
-      expect(mockedAxios.get).toHaveBeenCalled();
+      expect(<%= entityInstance %>ServiceStub.retrieve.called).toBeTruthy();
       expect(comp.<%= entityInstancePlural %>[0]).toEqual(jasmine.objectContaining({id: <%- tsKeyId %>}));
     });
 <%_ if (pagination !== 'no') { _%>
 
     it('should load a page', async () => {
       // GIVEN
-      mockedAxios.get.mockReturnValue(Promise.resolve({headers: {}, data: [{id: <%- tsKeyId %>}]}));
+      <%= entityInstance %>ServiceStub.retrieve.resolves({ headers: {}, data: [{ id: <%- tsKeyId %> }] });
 
       // WHEN
       comp.loadPage(1);
       await comp.$nextTick();
 
       // THEN
-      expect(mockedAxios.get).toHaveBeenCalled();
+      expect(<%= entityInstance %>ServiceStub.retrieve.called).toBeTruthy();
       expect(comp.<%= entityInstancePlural %>[0]).toEqual(jasmine.objectContaining({id: <%- tsKeyId %>}));
     });
     <%_ if (pagination !== 'infinite-scroll') { _%>
 
     it('should not load a page if the page is the same as the previous page', () => {
       // GIVEN
-      mockedAxios.get.mockReset();
+      <%= entityInstance %>ServiceStub.retrieve.reset();
       comp.previousPage = 1;
 
       // WHEN
       comp.loadPage(1);
 
       // THEN
-      expect(mockedAxios.get).toHaveBeenCalledTimes(0);
+      expect(<%= entityInstance %>ServiceStub.retrieve.called).toBeFalsy();
     });
     <%_ } _%>
 
     it('should re-initialize the page', async () => {
       // GIVEN
-      mockedAxios.get.mockReset();
-      mockedAxios.get.mockReturnValue(Promise.resolve({headers: {}, data: [{id: <%- tsKeyId %>}]}));
+      <%= entityInstance %>ServiceStub.retrieve.reset();
+      <%= entityInstance %>ServiceStub.retrieve.resolves({ headers: {}, data: [{ id: 123 }] });
```

Comment:
Why are you using `123` and not `tsKeyId` here?

Reply:
Good catch!